release-doc-in-process/all_vf.TTL

```diff
@@ -323,19 +323,57 @@ vf:containedIn a            owl:ObjectProperty ;
         rdfs:comment        "Used when a stock economic resource contains units also defined as economic resources." .
 
 vf:hasBeginning a           owl:DatatypeProperty ;
-        rdfs:domain         time:TemporalEntity ;
+        rdfs:domain         [ owl:unionOf (
+                                time:TemporalEntity
```

Comment:
Should remove "time:TemporalEntity"

Reply:
You don't think we still want to have that "tunnel" into owl time?

Reply:
OK then, i'm not religious about it :).